src/containers/ConceptPage/EditConcept.tsx

```diff
@@ -38,7 +40,7 @@ interface Props {
 
 const EditConcept = ({ isNewlyCreated }: Props) => {
   const params = useParams<'id' | 'selectedLanguage'>();
-  const conceptId = Number(params.id) || undefined;
+  const conceptId = Number(params.id);
```

Comment:
Hvorfor er denne endret? Det er ikke gitt at `params.id` er et gyldig tall. Vil gjerne unngå NaN

Reply:
Vet ikke, men tipper det kanskje ble gjort for å få kalt `updateConcept` uten undefined id som er litt dumt når den kan være undefined😅 Reverterer sånn at onUpdate er definert inline og ikke som callback, da vet vi at conceptId eksisterer i det den kalles